### ISSUE TITLE
multiple Lambda Functions in one project

### ISSUE THREAD

**Issue by Sipsip** [aws/aws-toolkit-eclipse#105]
I created a "Lambda Java Project" (including Handler Class A) via the toolkit and later I created another Lambda Function (Handler B) within that project.
Now when I run+upload Handler B it will change the configs on my aws-console, so that both Lambda Functions on aws cloud run the code of Handler A.

This seems to be an Issue within the file com.amazonaws.eclipse.lambda.project.json

Is it allowed to have multiple Handlers within one project and if yes, how do I solve my problem?

**Comment by zhangzhx** [comment]
Yes, it is allowed to have multiple Handlers in one project. I cannot reproduce this problem. Can you provide the version for your plugin?

**Comment by e4ustudio** [comment]
I face the same problem, when I create multiple functions within a AWS Lambda Java project. Each time I click "Upload function to AWS Lambda", all different functions are uploaded to the same AWS Lambda function. Hope this issue will be fixed soon!

**Comment by zhangzhx** [comment]
I still couldn't reproduce the issue. Did you select the handler correctly when uploading the new Lambda function to AWS?

**Comment by apathan123** [comment]
I am facing the similar issue. 
I had function 1 being run as schedules job successfully. I added new function in the same project. Everytime I upload + run function from eclipse it runs the new function even if I click on Test for the first function.  So I ended up spoiling the already working function. Will teh test alwasy trigger the latest one?
